Launch large project tutorial fix (#4454)
ros2/ros2_documentation: source/Tutorials/Intermediate/Launch/Using-ROS2-Launch-For-Large-Projects.rst

This launch file shows the way environment variables can be called inside the launch files.
Environment variables can be used to define or push namespaces for distinguishing nodes on different computers or robots.

> .. note:: If you running the launch file from an environment where `USER` is not defined like in the ROS docker (check with echo $USER), then you can substitute `EnvironmentVariable('USER')` with any other word of your liking.

Reviewer: ```suggestion
.. note:: If you are running the launch file where the `USER` environment variable is not defined (like in the ROS docker file), then you can replace the `EnvironmentVariable('USER')` above with any other word of your liking.
```
